### DISCUSSION TITLE
Swap A/B button

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#236] [Ideas]
Could we have a button to swap model set for ModelA and ModelB?
![изображение](https://github.com/hako-mikan/sd-webui-supermerger/assets/1073832/8978e666-93ec-4b55-adc5-cd25f8b721ea)

**Comment by wkpark** [comment]
I guess, you simply need to swap already selected models, right?
![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/f146e122-85a7-4e7d-8f8a-292c262618be)

**Reply by miasik** [reply]
You're right. Looks promising!

**Reply by miasik** [reply]
Is there any progress for the feature?
